libs/expo/betterangels/src/lib/screens/Client/Docs/UploadModal/index.tsx

```diff
@@ -0,0 +1,474 @@
+      ...docs,
+      DriversLicenseFront,
+      DriversLicenseBack,
+      SocialSecurityCard,
+      PhotoId,
+      BirthCertificate,
+      ConsentForm,
+      HmisForm,
+      IncomeForm,
+    });
+  }, [client]);
+
+  return (
+    <Modal
+      style={{
+        margin: 0,
+        flex: 1,
+        justifyContent: 'flex-end',
+      }}
+      animationIn="slideInUp"
+      animationOut="slideOutDown"
+      backdropOpacity={opacity}
+      isVisible={isModalVisible}
+      onBackdropPress={closeModal}
+      useNativeDriverForBackdrop={true}
+    >
+      <View
+        style={{
+          borderTopLeftRadius: Radiuses.xs,
+          borderTopRightRadius: Radiuses.xs,
+          paddingTop: topOffset + Spacings.xs,
+
+          backgroundColor: Colors.WHITE,
+          flex: 1,
+        }}
+      >
+        {tab ? (
+          TABS[tab]
+        ) : (
+          <ScrollView
+            style={{
+              paddingHorizontal: Spacings.sm,
+              paddingBottom: 35 + bottomOffset,
+            }}
+          >
+            <View
+              style={{
+                flexDirection: 'row',
+                alignItems: 'center',
+                justifyContent: 'space-between',
+                marginBottom: Spacings.sm,
+              }}
+            >
+              <TextBold size="lg">Upload Files</TextBold>
+              <Pressable
+                accessible
+                accessibilityHint="closes the modal"
+                accessibilityRole="button"
+                accessibilityLabel="close"
+                onPress={closeModal}
+              >
+                <PlusIcon size="md" color={Colors.BLACK} rotate="45deg" />
+              </Pressable>
+            </View>
+            <TextRegular size="sm" mb="md">
+              Select the right file type and you can rename it when it's done
+              (optional).
+            </TextRegular>
+            <View style={{ gap: Spacings.xs, marginBottom: Spacings.lg }}>
+              <TextBold>Doc-Ready</TextBold>
+              <View
+                style={{
+                  flexDirection: 'row',
+                  alignItems: 'center',
+                }}
+              >
+                <View
+                  style={{
+                    height: 20,
+                    width: 20,
+                    borderRadius: Radiuses.xxxl,
+                    backgroundColor:
+                      !!docs.DriversLicenseFront && !!docs.DriversLicenseBack
+                        ? Colors.SUCCESS
+                        : Colors.NEUTRAL_LIGHT,
+                    marginRight: Spacings.xs,
+                    alignItems: 'center',
+                    justifyContent: 'center',
+                  }}
+                >
+                  {!!docs.DriversLicenseFront && !!docs.DriversLicenseBack && (
+                    <CheckIcon size="sm" color={Colors.WHITE} />
+                  )}
+                </View>
+
+                <Button
+                  disabled={
```

Comment:
I know this logic is consistent with the [design](https://www.figma.com/design/waIeX8dknHKedTrVxCQAAG/HIFI---1st-MVPs?node-id=14541-93420&t=4qSbzIWUBRl80Xjw-4), but I think we should change it for the MVP. My reasoning is: this makes it impossible to upload a new version of a document before deleting the existing version. But the MVP doesn't include "delete" functionality, so this would make it impossible to correct a mistake.

Reply:
don't make this change yet, actually. Let's check with prod. Because we were trying to scope down the MVP for the SELAH deadline, but there is no deadline anymore